homeassistant/components/tado/const.py

```diff
@@ -48,6 +48,12 @@
 DATA = "data"
 UPDATE_TRACK = "update_track"
 
+CONFIGURATION_URL_HOME = "https://app.tado.com"
+CONFIGURATION_URL_ZONE = "https://app.tado.com/en/main/home/zoneV2/{}"
```

Comment:
Alternatively, this could link to `/settings/rooms-and-devices/room/{}` but I thought that changing a schedule is a more common task and it's accessible from the room page but not from settings.